docker/Dockerfile.x86_64-unknown-linux-gnu

```diff
@@ -22,8 +16,9 @@ RUN /qemu.sh x86_64 softmmu
 COPY dropbear.sh /
 RUN /dropbear.sh
 
-COPY --from=0 /qemu /qemu
+COPY linux-image.sh /
+RUN /linux-image.sh x86_64
 
 COPY linux-runner /
 
-ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER="/linux-runner x86_64"
+ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER="/linux-runner x86_64"
```

Comment:
I don't think this change was intentional, also, someone's text editor will automatically add this newline.